commands.js

```diff
@@ -744,6 +745,7 @@ exports.commands = {
 
 		delete room.auth[userid];
 		this.sendReply("(" + name + " is no longer Room Owner.)");
+		targetUser.popup("You are no longer Room Owner of " + room + ". (Demoted by " + user.name + ".)");
```

Comment:
This would crash if <code>targetUser</code> did not exist (see relevant line below).